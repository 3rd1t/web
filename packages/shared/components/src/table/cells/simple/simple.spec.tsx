import React from "react"
import { render } from "@testing-library/react"

import Simple from "./simple"

describe("Simple", () => {
  it("should render successfully", () => {
    const { baseElement } = render(<Simple />)
    expect(baseElement).toBeTruthy()
  })
})
