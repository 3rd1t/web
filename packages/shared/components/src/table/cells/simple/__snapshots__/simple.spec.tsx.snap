// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Simple Stories primaryshould match snapshot 1`] = `
<body>
  <div>
    <div
      class="px-6 py-4 whitespace-no-wrap"
    >
      <span
        class="text-sm leading-5 text-gray-900"
      >
        Label
      </span>
    </div>
  </div>
</body>
`;
